src/Utilities/tests/CMakeLists.txt

```diff
@@ -23,3 +23,16 @@ TARGET_LINK_LIBRARIES(${UTEST_EXE} qmcutil ${QMC_UTIL_LIBS} ${MPI_LIBRARY})
 
 #ADD_TEST(NAME ${UTEST_NAME} COMMAND "${QMCPACK_UNIT_TEST_DIR}/${UTEST_EXE}")
 ADD_UNIT_TEST(${UTEST_NAME} "${QMCPACK_UNIT_TEST_DIR}/${UTEST_EXE}")
+
+IF(HAVE_MPI)
+  SET(UTEST_EXE test_${SRC_DIR}_mpi)
+  SET(UTEST_NAME deterministic-unit_test_${SRC_DIR}_mpi)
+  #this is dependent on the directory creation and sym linking of earlier driver tests
+  SET(MPI_UTILITY_TEST_SRC test_mpi_exception_wrapper.cpp)
+  ADD_EXECUTABLE(${UTEST_EXE} ${MPI_UTILITY_TEST_SRC})
+  USE_FAKE_RNG(${UTEST_EXE})
```

Comment:
I don't think we need this RNG stuff here.

Reply:
Will delete that